test/runtests.jl

```diff
@@ -20,6 +20,7 @@ bsrs_regex = ReplicateDesign(bsrs.data, REPLICATES_REGEX, weights = :pw)  # usin
 # Stratified sample
 apistrat = load_data("apistrat") # Load API dataset
 dstrat = SurveyDesign(apistrat, strata = :stype, weights = :pw) # Create SurveyDesign
+dstrat_boot = dstrat |> bootweights # Create replicate design
```

Comment:
```suggestion
```

Reply:
So we don't need to add replicate weights for stratified sampling?

Reply:
This is already there `bstrat` in line 24